### PR TITLE
feat(it): allow passing in extra args to the start command

### PR DESCRIPTION
**Pull request by fieldju**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ inputs:
   deployHostUrl:
     description: 'Location of the Armory Cloud API which will handle your deployment'
     default: 'api.cloud.armory.io'
+  extraArgs:
```
Comment (tunderwood) [inline action.yaml:22]: Do we want to leave this generic and call it extra args or do we want to say "name" and try to compose the command. This is easier but is, in my opinion, a less ideal user experience.

Comment (fieldju) [reply action.yaml:22]: Yes, but we will need an entry-point.sh, I was being lazy, I will update PR.

Comment (fieldju) [reply action.yaml:22]: - [ ] make this nicer ;P

**Comment by fieldju** [comment]
I'm gonna make an RC tag off of this branch to test it, with our pipelines

**Review by tunderwood — APPROVED** [review]

**Review by th3morg — APPROVED** [review]

**Pull request merged**